SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,22 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
```

Comment:
Regardless of the outcome of the discussion about how they'll work, we should include some details here about what's happening with respect to user accounts and the forums.
If we're doing the same as normal then we should encourage the team leaders to have created their teams' accounts. Otherwise we should include info on what the plan is.